http://bugzilla.openvz.org/show_bug.cgi?id=837

From: Alexey Dobriyan <adobriyan@openvz.org>
Date: Mon, 3 Mar 2008 12:02:38 +0000 (+0300)
Subject: SLUBC: change SLAB_UBC, SLAB_NO_CHARGE flags
X-Git-Url: http://git.openvz.org/?p=linux-2.6.24-openvz;a=commitdiff_plain;h=73980b5b87e7619ba49364d2618b25d4f4bd8450

SLUBC: change SLAB_UBC, SLAB_NO_CHARGE flags

SLAB_NO_CHARGE conflicted with __SYSFS_ADD_DEFERRED internal flag
which resulted in double and broken additional directories created
in /sys/slab/ .
---

diff --git a/include/linux/slab.h b/include/linux/slab.h
index 2ff029a..793613f 100644
--- a/include/linux/slab.h
+++ b/include/linux/slab.h
@@ -62,8 +62,8 @@
  *				        poll, fdsets, ...)  non-ub allocs)
  * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  */
-#define SLAB_UBC		0x20000000UL	/* alloc space for ubs ... */
-#define SLAB_NO_CHARGE		0x40000000UL	/* ... but don't charge */
+#define SLAB_UBC		0x10000000UL	/* alloc space for ubs ... */
+#define SLAB_NO_CHARGE		0x20000000UL	/* ... but don't charge */
 
 /*
  * struct kmem_cache related prototypes
